src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/AnnotationStub.java

```diff
@@ -26,6 +27,25 @@ public AnnotationEntity createAndSaveAnnotationEntityWith(UserAccountEntity curr
         return annotationRepository.save(annotationEntity);
     }
 
+    @Transactional
+    public AnnotationEntity createAndSaveAnnotationEntityWith(UserAccountEntity currentOwner,
+                                                              String annotationText,
+                                                              HearingEntity hearingEntity) {
+        AnnotationEntity annotationEntity = createAnnotationEntity(currentOwner, annotationText, hearingEntity);
+        return annotationRepository.save(annotationEntity);
+    }
+
+    public static AnnotationEntity createAnnotationEntity(UserAccountEntity currentOwner, String annotationText, HearingEntity hearingEntity) {
```

Comment:
No it can't be because the way the entity is setup with persist and merge, you cannot add the annotation to the annotation document and then to the eod as you get a jpa exception. By changing the annotation document to not have persist and merge would require the AnnotationService to be rewritten. So as I just need the object I created it the way it is now.